R/ResampleDesc.R

```diff
@@ -167,6 +190,22 @@ print.RepCVDesc = function(x, ...) {
   catf("Stratification: %s", x$stratify)
 }
 
+#' @export
+print.GrowingCVDesc = function(x, ...) {
+  catf("Window description:\n %s: %.2f %s observations in initial window and %i horizon.",
```

Comment:
I could not figure out how to add a percent sign in the output so that is why there is a %s. The output would look like

```
Window Description:
20% of observations in initial window and 1 horizon
``